extractor/src/main/scala/com/digitalasset/extractor/config/ConfigParser.scala

```diff
@@ -279,7 +279,7 @@ object ConfigParser {
         cliParams.ledgerPort,
         from,
         to,
-        cliParams.party,
+        ExtractorConfig.parties(cliParams.party),
```

Comment:
@leo-da This is the splitting you were looking for; I have a weak preference for doing as little parsing into `CliParams` as possible, but do you want me to add _moar type safety_ for `CliParams`?

Reply:
nah, it is fine... but my main concern was documentation, the help screen. Plus the thing that @bitonic wanted you need `unbounded` option configuration, to support multiple party options.

Reply:
Docced in 1c2cb89d24